docs/docs/reference/custom-components/05-aica-package-toml.md

```diff
@@ -39,112 +39,137 @@ Each `aica-package.toml` should start with `#syntax=ghcr.io/aica-technology/pack
 
 ```toml title="aica-package.toml"
 [build]
 type = "ros"
 ```
 
-#### `[build.cmake_args]`
+#### `[build.ssh]`
 
-Optional. In this category, you can specify arguments which will be passed to CMake through `-DK1=V1 -DK2=V2 ...`
+Optional. Default is `false`. If set to `true`, any call to `CMake`, `pip` or custom stage will have access to the SSH credentials given to Docker with `--ssh default`. This is useful if you need to clone a private repository in CMakeLists.txt or install a private Python package from git.
 
 ```toml title="aica-package.toml"
-[build.cmake_args]
-# here are a few examples
-PACKAGE_NAME = "my_package"
-USE_FEATURE_X = "ON"
+[build]
+ssh = false
 ```
 
-#### `[build.environment]`
-
-Required. This category describes the environment (ROS version, libraries, etc.) that will be used to build the
-components.
+#### `[build.image]`
 
-`[build.environment.ssh]` is optional (defaults to `false`). If set to `true`, any call to `CMake`, `pip` or custom
-stage will have access to the SSH credentials given to Docker with `--ssh default`. This is useful if you need to clone
-a private repository in CMakeLists.txt or install a private Python package from git.
+Required. `[build.image]` is the tag of the AICA `ghcr.io/aica-technology/ros2-ws` image that will be used to build the components. Those images are tagged after the versions of the ROS 2 distributions and are available [here](https://github.com/aica-technology/docker-images/pkgs/container/ros2-ws).
 
 ```toml title="aica-package.toml"
-[build.environment]
-ssh = false
+[build]
+image = "iron"
 ```
 
-`[build.environment.aica]` is required. `[build.environment.aica.image]` is the tag of the
-AICA `ghcr.io/aica-technology/ros2-control` image that will be used to build the components. Those images are tagged
-after the ROS2 distributions and are
-available [here](https://github.com/aica-technology/docker-images/pkgs/container/ros2-control).
+#### `[build.cmake-args]`
+
+Optional. In this category, you can specify arguments which will be passed to CMake through `-DK1=V1 -DK2=V2 ...`
 
 ```toml title="aica-package.toml"
-[build.environment.aica]
-image = "iron"
+[build.cmake-args]
+# here are a few examples
+PACKAGE_NAME = "my_package"
+USE_FEATURE_X = "ON"
 ```
 
-##### `[build.environment.aica.libraries]`
+#### `[build.dependencies]`
 
-Optional. `[build.environment.aica.libraries]` is used to specify the AICA libraries that will be installed in the
+Optional. `[build.dependencies]` is used to specify the AICA libraries and ROS 2 packages that will be installed in the
 image.
 
 :::note
-Those libraries will not be available at runtime, only while building. If you need them for building _and_ running, add
-them to the `[build.packages.XYZ.dependencies]` section below instead.
+Those libraries and packages will not be available at runtime, only while building. If you need them for building _and_ running, add them to the `[build.packages.XYZ.dependencies]` section below instead.
```

Comment:
Might be nice to add a note here along the lines of "Libraries that are only required at runtime, and not at build time, can be declared with `build.run-dependencies` instead"